ajax/libs/vast-client/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "vast-client",
+  "filename": "vast-client.min.js",
+  "description": "Javascript VAST Client",
+  "version": "1.8.0",
+  "keywords": [
+    "vast",
+    "ad",
+    "advertising",
+    "iab",
+    "in-stream",
+    "video"
+  ],
+  "author": "Olivier Poitrey <rs@dailymotion.com>",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/dailymotion/vast-client-js"
```

Comment:
Let's modify as the same with upstream.
`"url": "https://github.com/dailymotion/vast-client-js"`
https://github.com/dailymotion/vast-client-js/blob/v1.8.0/package.json#L16
Thanks